_posts/2022-07-13-announcing-enmeshed-v2-requests.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "Announcing Enmeshed v2 Requests"
+date: 2022-07-13T1:00:00+02:00
+categories:
+    - blog
+tags:
+    - announcement
+    - v2
+    - Requests
+toc: true
+---
+
+This is one of the blog posts regarding Enmeshed v2. For an overview of all Enmeshed v2 blog posts, please refer to the [Enmeshed v2 announcement blog post]({% post_url 2022-06-27-announcing-enmeshed-v2 %}).
+
+This blog post requires a superficial understanding of the new Attribute handling. Please refer to the [corresponding blog post]({% post_url 2022-06-30-announcing-enmeshed-v2-attributes %}) to learn more about it if you are not yet familiar with it.
+
+This blog post describes what Requests are in the Enmeshed universe and how they are used to exchange Attributes and establish Relationships.
+
+When we mention "the App" or "the Connector" in this blog post, we mean the official [Enmeshed App]({% link _docs_explore/50-app.md %}) and [Enmeshed Connector]({% link _docs_explore/52-connector.md %}).
+
+## Requests
+
+Requests in Enmeshed always defined a way to exchange structured data. In Enmeshed V1 this was exclusively about Attributes. Requests in V2 will also be able to exchange unstructured data without persisting it to the User's Attributes. One example is a form you can send to a User, which contains some questions in natural language.
```

Comment:
Requests in Enmeshed always defined a way to exchange structured data. In Enmeshed V1 this was exclusively about exchanging structured Attributes with its `AttributesChangeRequest` and `AttributesShareRequest`. In V1 there also was no defined response structure, as well as no track record of Requests and their status. Thus, V1 Requests were quite limited.

Requests in V2 extend the Request featureset by providing more structured Requests, as well as Requests for unstructured data. Additionally, they are not fixed to the Identitity's Attributes. One example is a form you can send to a user, which contains some questions in natural language, which does not affect the Attributes. Another example is a Multi-Factor-Authentication Request which might be available in the future.